csv.py

```diff
@@ -0,0 +1,203 @@
+import math
+import sys
+from volatility import renderers
+from volatility.fmtspec import FormatSpec
+from volatility.renderers import ColumnSortKey
+from volatility.renderers.basic import Address, Address64, Hex, Renderer
+
+__author__ = 'david'
+
+class CellRenderer(object):
+    """Class to handle rendering of a particular cell in a text grid"""
+    # The minimum width that the renderer will produce for a value
+    width = 0
+
+    def render(self, value):
+        """Returns the rendering of an individual value"""
+        return value
+
+
+class FormatCellRenderer(CellRenderer):
+    """Class to handle rendering each cell of a grid"""
+
+    def __init__(self, format_spec):
+        if not isinstance(format_spec, FormatSpec):
+            fs = FormatSpec()
+            fs.from_string(format_spec)
+            format_spec = fs
+        self._format_spec = format_spec
+
+    def render(self, value):
+        """Render an individual cell"""
+        return ("{0:" + str(self._format_spec) + "}").format(value)
+
+    @property
+    def width(self):
+        return self._format_spec.minwidth
+
+    @width.setter
+    def width(self, value):
+        self._format_spec.minwidth = max(value, self._format_spec.minwidth)
+
+    def __repr__(self):
+        return "<FormatCellRenderer (" + repr(self._format_spec) + ")>"
+
+
+class TextRenderer(Renderer):
```

Comment:
I would suggest is importing the TextRenderer() class instead of redefining it here.

Reply:
I had made a pull request for commands.py too.
I just reuse the greptext rendere to make my csv rendere.

I think that it's possible to upgrade the greptext rendere too with : join "|" and not a +=